extensions/src/test/java/org/pentaho/platform/osgi/KarafBootTest.java

```diff
@@ -382,6 +383,21 @@ public void testClearDataCacheSetting() throws Exception {
 
   }
 
+  @Test
+  public void deleteRecursiveIfExistsTest() {
+
+    File dir = new File( "dir" );
+    dir.mkdir();
+    File file = new File( dir,  "file" );
+    try {
+      file.createNewFile();
+      KarafBoot.deleteRecursiveIfExists( dir );
+      assertNull( dir.listFiles() );
+    } catch ( IOException e ) {
+      e.printStackTrace();
```

Comment:
Should not have a printStackTrace here. If you get an IOException, shouldn't the test fail?